### DISCUSSION TITLE
How do I setup my financial journey with Paisa

### DISCUSSION THREAD

**Discussion by Mozenraath** [ananthakumaran/paisa#135] [Q&A] [answered]
End of 2022, I started using Mint to track my expenses and incomes. 2023 was my first year where I could see the overall trend and summary of my expenditures. Since Mint is closing down by April this year (moving to credit karma), I started looking for alternatives and came across Paisa (wonderful tool). Since last  2 weeks I am trying to re-create my financial transactions dating back to 2018 but all data is not available in terms of expenditures, transactions between accounts and so on.  In between I purchased the house, incomes from Expats, max gain home loan account, mutual funds.... and this makes it very difficult for me to setup accurate version of my financial history.
I am just looking for some help how I can setup or what can be my starting point (or maybe it doesn't matter)... Anyone who went through similar experience, can share their insights. That would be great !!
@ananthakumaran Really appreciate your efforts and work ! Paisa is beautiful program.

**Comment by ananthakumaran** [comment, marked as answer]
I initially started with just Income/Assets and marked unknown expenses as Expenses:Misc. This way, I would know how much money I got and how much I saved, and the spent part wouldn't have a breakdown. Later I gradually start to track with more granularity.

Ledger just expects the transaction to balance and you can always start with very coarse transactions and later make it more granular if needed.

**Reply by Mozenraath** [reply]
Important thing is I got the setup done for this year. Still learning the basics of plain text accounting. not fully there. hopefully as I get time and patience would slowly build my past data too.

**Comment by swarajsomala** [comment]
A few months ago, I was in the same situation, but I found a solution that works for me. I made a separate file called `legacy.ledger` to keep track of my investments, insurance, and loan repayments from the past three years. I imported the details using the Paisa feature, made some changes, and labeled the income as legacy salary.
Here is the sample:
```
    
2020/11/01 Loan Repayment  
    Liabilities:PersonalLoan                      1000 INR
    Liabilities:CarLoan                              100   INR
    Expenses:Interest:CarLoan                 1000 INR
    Expenses:Interest:PersonalLoan         100 INR
    Expenses:Insurance:Life                     100 INR
    Income:Salary:Legacy
```

I added the `legacy.ledger` file to the main ledger, and now I can view all the details in the Paisa UI. My legacy salary might not match the actual amount I received, as I spent money on things I can't remember now. However, this way, I can at least estimate that some of my income was used for something specific that I can track. If I don't want to see the legacy details, I can simply comment out the import of the `legacy.ledger`, displaying only the current details.

**Reply by Mozenraath** [reply]
I am more less following the same way however some of the data is just trying to get today's balance right.
